### ISSUE TITLE
GPWP tries to include json files from externals

### ISSUE THREAD

**Issue by pixelplanetdev** [lostpebble/generate-package-json-webpack-plugin#37]
I import .json configuration files that my application is supposed to read at launch, so i included them in the webpack externals to not get them bundled
```
    externals: [
      /\/proxies\.json$/,
      /\/canvases\.json$/,
      /^\.\/styleassets\.json$/,
      /^\.\/assets\.json$/,
      nodeExternals(),
    ],

```
but the GPJWP plugin, after a recent update, wants to include them and that gives some warnings:

```
GPJWP: Ignoring module without a found package.json: . ("package.json" couldn't resolve): Cannot find module '/home/andreas/Projekte/pixelplanet/server.js'. Please verify that the package.json has a valid "main" entry
GPJWP: Couldn't resolve a version for module "." (from portable ID: external commonjs2 "./canvases.json") - will be ignored in final output.
GPJWP: Ignoring module without a found package.json: . ("package.json" couldn't resolve): Cannot find module '/home/andreas/Projekte/pixelplanet/server.js'. Please verify that the package.json has a valid "main" entry
GPJWP: Couldn't resolve a version for module "." (from portable ID: external commonjs2 "./assets.json") - will be ignored in final output.
GPJWP: Ignoring module without a found package.json: . ("package.json" couldn't resolve): Cannot find module '/home/andreas/Projekte/pixelplanet/server.js'. Please verify that the package.json has a valid "main" entry
GPJWP: Couldn't resolve a version for module "." (from portable ID: external commonjs2 "./styleassets.json") - will be ignored in final output.
GPJWP: Ignoring module without a found package.json: . ("package.json" couldn't resolve): Cannot find module '/home/andreas/Projekte/pixelplanet/server.js'. Please verify that the package.json has a valid "main" entry
GPJWP: Couldn't resolve a version for module "." (from portable ID: external commonjs2 "./proxies.json") - will be ignored in final output.
```

This didn't happen before, so i thought that might be a bug :)

**Comment by pixelplanetdev** [comment]
//cc @lostpebble

**Comment by lostpebble** [comment]
Thanks @pixelplanetdev - I've published your changes to 2.6.0

**Comment by pixelplanetdev** [comment]
awesome, thanks
